### PR TITLE
Update the groundwater_daily_value id column to bigint to accomodate …

### PR DESCRIPTION
**Pull request by ssoper-usgs**
…our frequent DELETE/INSERT strategy

### CODE REVIEW

File: liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml [hunk 1]
```diff
@@ -50,3 +50,12 @@ databaseChangeLog:
...
         - rollback: alter table ${NWIS_SCHEMA_NAME}.groundwater_daily_value drop column updated_at;
+
+  - changeSet:
+      author: ssoper
+      id: "alter.table.${NWIS_SCHEMA_NAME}.groundwater_daily_value.alter.column.groundwater_daily_value_id.type.bigint"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+      changes:
+        - sql: alter table ${NWIS_SCHEMA_NAME}.groundwater_daily_value alter column groundwater_daily_value_id type bigint;
```
Comment (kkehl-usgs) [inline liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml:61]: rollback needed?

**Review by kkehl-usgs — APPROVED** [review]

File: liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml (hunk 1 above)
Comment (ssoper-usgs) [reply liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml:68]: In this case I don't think we need it - postgres11 should give that to us for free if the update fails.  I think @orosz-usgs had more details on this design paradigm.

Comment (orosz-usgs) [reply liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml:68]: Our thinking on the rollback has changed. Too much to go into on a Friday. Can go over later.

**Review by orosz-usgs — APPROVED** [review]

File: liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml (hunk 1 above)
Comment (orosz-usgs) [reply liquibase/changeLogs/nwis/tables/groundwaterDailyValue/changeLog.yml:68]: one reason is that the operation should be atomic. If there was an error, no update was done.  Nothing really
to rollback.

**Pull request merged**